packages/table/src/Row/Row.spec.tsx

```diff
@@ -58,27 +57,28 @@ const RowWithNestedRows = args => {
                     </Cell>
                   );
                 })}
-                {row.subRows &&
-                  row.subRows.map(subRow => (
-                    <Row key={subRow.id} row={subRow}>
-                      {subRow.getVisibleCells().map(cell => {
-                        return (
-                          <Cell key={cell.id}>
-                            {flexRender(
-                              cell.column.columnDef.cell,
-                              cell.getContext(),
-                            )}
-                          </Cell>
-                        );
-                      })}
-                    </Row>
-                  ))}
               </Row>
```

Comment:
Q - For my own context: Is this the API we currently have, where sub-rows are siblings and not children?

Reply:
Correct, and the new API is not actually reflected in my changes to file. I'm going to create a new PR updating all the specs.